page-officers.hbs

```diff
@@ -332,6 +332,15 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-github" href="https://github.com/DarKnight0102/"></a>
                   </div>
                 </div>
+                <div class="ieee-officer-card">
+                  <img src="{{asset 'officers/Website/andrew_zhang.jpg'}}" alt="profilepic"/>
```

Comment:
You're going to have to turn this image into a square as it renders incorrectly on the page due to its current dimensions.